www/shared/globals.py

```diff
@@ -31,3 +36,7 @@
 }
 
 local_prefix = os.getenv('LOCAL_PREFIX', './tmp')
+
+SECRET_KEY = os.getenv('SECRET_KEY', 'Super Scret Key For Flask App')
```

Comment:
I think all of this can be manage by `jwt_settings` to continue with the combention